indexing-service/src/main/java/org/apache/druid/indexing/common/task/batch/parallel/PartialSegmentMergeTask.java

```diff
@@ -266,6 +267,12 @@ private Set<DataSegment> mergeAndPushSegments(
             persistDir,
             0
         );
+        long mergeFinishTime = System.nanoTime();
+        LOG.info("Merging [%d] input segment(s) for interval [%s] took [%,d]ms.",
```

Comment:
Sorry for the really nitty nit, but this message initially made me think that it was in the future rather than past tense.  What do you think about making this

```
"Merged [%,d] input segments for interval [%s] in [%,d]ms"
```